lib/ejs.js

```diff
@@ -381,6 +381,10 @@ exports.render = function (template, d, o) {
     utils.shallowCopyFromList(opts, data, _OPTS);
   }
 
+  if (opts.fileLoader) {
+    fileLoader = opts.fileLoader; // eslint-disable-line
```

Comment:
`fileLoader` isn't declared anywhere; why are you assigning to an undeclared variable?

Reply:
Actually, it is being declared as a function here: https://github.com/mde/ejs/blob/master/lib/ejs.js#L239

Reply:
That still won't work correctly, because after passing fileloader as an option, you won't be able to fall back to `ejs.fileLoader` on a future function call.

Reply:
@RyanZim you're right, totally missed that. I'll be refactoring it to take that into account as soon as possible.